### ISSUE TITLE
Vader Immortal crashes onload

### ISSUE THREAD

**Issue by dgatto** [LibreVR/Revive#1624]
Valve Index
AMD Ryzen 7
GTX 2080 Super
Windows 10
16 GB RAM

Vader Immortal crashes on load when it's still loading with a black screen. Can't find any error messages unless they're in a log file somewhere.

Other games, such as Superhot, do work from the Revive Dashboard.

Revive is installed to my C:/ProgramFiles where my Operating System and all other Program Files are.
Vader Immortal is installed to my D:/Games where all of my games are.

**Comment by dgatto** [comment]
Uhm, I just tried starting Vader Immortal without starting SteamVR first and then it booted up fine. Anyone finding this in the future, try letting Revive start your SteamVR. That may work.

**Comment by dgatto** [comment]
Okay, even more odd:
I booted Vader Immortal without steamVR started and it had controller problems, so I shut it down and tried starting SteamVR first again, then it worked!

So what I had to do was:
1. Boot up Vader Immortal without starting SteamVR first,
2. Let it complain about not being able to find controllers.
3. Close Vader Immortal
4. Start SteamVR
5. Start Vader Immortal from Revive Dashboard
6. Any bootups from Revive with SteamVR already running should work afterwards.

Really weird, idk how I got it to work. I'm just trying to document everything so that anyone else coming in the future can possibly use this to get theirs working.
